hrui/scss/_variables.scss

```diff
@@ -0,0 +1,8 @@
+// Colors
```

Comment:
I believe you should be reusing the existing variables from org.civicrm.bootstrapcivicrm